packages/airnode-validator/src/config/config.test.ts

```diff
@@ -69,3 +69,71 @@ describe('nodeSettingsSchema', () => {
     );
   });
 });
+
+it('fails if ois.apiSpecifications.security.<securitySchemeName> is defined in ois.apiSpecifications.components.<securitySchemeName> but apiCredentials.securitySchemeValue is empty string', () => {
+  const config: Config = JSON.parse(
+    readFileSync(join(__dirname, '../../test/fixtures/interpolated-config.valid.json')).toString()
+  );
+
+  const securitySchemeName = 'Currency Converter Security Scheme';
+  const securityScheme = {
+    securitySchemeName: {
+      in: 'query',
+      type: 'apiKey',
+      name: 'access_key',
+    },
+  };
+  const apiCredentials = [
+    {
+      oisTitle: 'Currency Converter API',
+      securitySchemeName: 'Currency Converter Security Scheme',
+      securitySchemeValue: '${SS_CURRENCY_CONVERTER_API_KEY}',
+    },
+  ];
+  const configWithSecuritySchemes = {
+    ...config,
+    ois: [
+      ...config.ois,
+      {
+        ...config.ois[0],
+        apiSpecifications: {
+          ...config.ois[0].apiSpecifications,
+          components: { securitySchemes: securityScheme },
+          security: { [securitySchemeName]: [] },
+        },
+      },
+    ],
+    apiCredentials,
+  };
+
+  const invalidConfigEmtpyApiCredentialValue = {
```

Comment:
```suggestion
  const invalidConfigEmptyApiCredentialValue = {
```

Reply:
But I'd just call this `invalidConfig` and test that the value is missing in `apiCredential`.